### PR TITLE
add settings checkbox to en/dis-able notifications

### PR DESCRIPTION
**Pull request by kaefert**

### CODE REVIEW

File: app/src/main/res/xml/root_preferences.xml [hunk 1]
```diff
@@ -8,4 +9,9 @@
             app:title="@string/theme"
             app:useSimpleSummaryProvider="true" />
     </PreferenceCategory>
+    <CheckBoxPreference
+        android:defaultValue="false"
+        android:key="show_notifications"
+        android:summary="for debugging, independent of TRIfA"
```
Comment (zoff99) [inline app/src/main/res/xml/root_preferences.xml:15]: can you make that translate able?

File: app/src/main/res/xml/root_preferences.xml [hunk 2]
```diff
@@ -8,4 +9,9 @@
...
+        android:title="Show notifications" />
```
Comment (zoff99) [inline app/src/main/res/xml/root_preferences.xml:16]: can you make that translate able?

Comment (kaefert) [reply app/src/main/res/xml/root_preferences.xml]: okey. moved those two strings to strings.xml

**Comment by kaefert** [comment]
tested & made it actually work with last commit ("bugfix nullpointer" = [44da36a](https://github.com/zoff99/tox_push_msg_app/pull/8/commits/44da36ace3ed22420572b337cd0acbc52264655f))

**Review by zoff99 — APPROVED** [review]

**Review by zoff99 — APPROVED** [review]

**Pull request merged**